src/semgrep_agent/main.py

```diff
@@ -87,7 +88,7 @@ def url(string: str) -> str:
     "--publish-deployment",
     envvar="INPUT_PUBLISHDEPLOYMENT",
     type=int,
-    help="You semgrep.dev organization ID (requires --publish-token)",
+    help="Your semgrep.dev organization ID (requires --publish-token)",
```

Comment:
Let's call this `deployment ID` instead of `organization ID` to match the new settings panel.

Reply:
Should this also change on line 85? Currently:
`Your semgrep.dev API token (only needed if specifying a publish organization)`

Reply:
Ooh, yeah, I think "publish deployment" or "deployment ID" instead of "publish organization" - having two terms is already confusing enough 😅